src/atoms/button-link/index.tsx

```diff
@@ -37,8 +38,14 @@ const ButtonLink = (props: ButtonLinkProps) => {
     dataTestId = '',
     eventId = '',
     style,
+    rel = undefined,
   } = props;
 
+  const getRelationshipAttributes = () => {
+    if (rel && rel !== '') return rel;
```

Comment:
I think we can always put `noopener noreferrer` if it has `target="_blank"` and ignore some custom `rel` attribute in that case for security reasons. What do you think?